src/databricks/labs/remorph/reconcile/connectors/data_source.py

```diff
@@ -45,8 +51,9 @@ def _get_jdbc_reader_options(options: JdbcReaderOptions):
             "fetchsize": options.fetch_size,
         }
 
-    def _get_secrets(self, key):
-        return self.ws.secrets.get_secret(self.scope, self.source + '_' + key)
+    def get_secrets(self, key_name: str):
```

Comment:
```suggestion
    def _get_secrets(self, key_name: str):
```
Method has to remain protected - you can still use it in subclasses, but not outside of them